bookmark.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BOOKMARK_H
+#define BOOKMARK_H
+#include <string>
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+class Bookmark {
+        public:
+            void displayMainMenu();
+            void addUser(int userCounter);
```

Comment:
this is not clear enough. addUser should accept the user object not counter as a param. fix the occurrences below